crates/fuel-core/src/service/adapters.rs

```diff
@@ -60,7 +60,6 @@ pub struct BlockProducerAdapter {
 pub struct BlockImporterAdapter {
     pub block_importer:
         Arc<fuel_core_importer::Importer<Database, ExecutorAdapter, VerifierAdapter>>,
-    execution_semaphore: Arc<tokio::sync::Semaphore>,
```

Comment:
I removed the semaphore from here for now because `Importer` already returns an error if the import process is in progress. 

We don't plan to import several blocks simultaneously. I think receiving an error is better than hiding it under the semaphore.

Reply:
Would this be a problem in local testing if manual-block-production is enabled? I.e. maybe they have interval mode set for block time but manually trigger some blocks and there's a conflict.

Reply:
If they face an error in the tests, it may be better to update the test=D Because if you produce more blocks during manual production than you asked, it may cause wrong test results.